iphone/Classes/TiAppiOSProxy.m

```diff
@@ -955,6 +941,27 @@ - (TiAppiOSLocalNotificationProxy *)scheduleLocalNotification:(id)args
   }
 }
 
+/**
+ Assign custom user info to notification content and make sure it has an identifier.
+
+ @param userInfo User info dictionary to assign to the notification content
+ @param content Notification content, can either be UNNotificationContent or UILocalNotification
+ @param notificationIdentifier The unique idenitifer for a notification.
+ */
+- (void)assignUserInfo:(NSDictionary *)userInfo toContent:(id)content ensureIdentifier:(NSString)notificationIdentifier
```

Comment:
NSString should be  NSString * . It is causing build issue. There are other build issues as well please check.